src/components/app/filter-dropdown.jsx

```diff
@@ -243,6 +243,17 @@ class FilterDropdown extends Component {
     return (<FilterCategory noSearch items={items}/>);
   };
 
+  renderReviews = () => {
+    const filters = getFilters();
+    const {reviews} = filters.getState();
+
+    const items = ['my-reviews', 'reviews-under-my-pr', 'my-meta-reviews', 'others'].map((review) => {
+      return {text: review, isSelected: reviews.indexOf(review) >= 0, toggleHref: filters.toggleReviews(review).url() };
```

Comment:
split into multiple lines?

also noting that these methods are horribly repetitive. (i.e. renderTypes / renderStates / etc), but that should get fixed in a separate PR before or after this one, maybe by someone else ;-)